core/src/nusi/fixed-select.tsx

```diff
@@ -17,9 +17,13 @@ import { get } from 'lodash';
 
 const { Option, OptGroup } = Select;
 const FixedSelect = React.forwardRef((props, ref) => {
-  const options = props.children || get(props, 'options');
+  const { options: propsOptions, ...rest } = props;
+  
+  //Determine if the child elements of props.options is components
+  const isComponent = props.options?.find(item => item.type?.name === 'Option');
+  const options = isComponent ? props.children || get(props, 'options') : '';
   return (
-    <Select ref={ref} getPopupContainer={(triggerNode) => triggerNode.parentElement as HTMLElement} {...props}>
+    <Select ref={ref} getPopupContainer={(triggerNode) => triggerNode.parentElement as HTMLElement} {...(isComponent && rest || props)}>
```

Comment:
prettier not working?

Reply:
I thought my prettier works when I save but it wasn't.